sha2/README.md

```diff
@@ -7,12 +7,52 @@
 [![Project Chat][chat-image]][chat-link]
 [![Build Status][build-image]][build-link]
 
-Pure Rust implementation of the [SHA-2 hash function family][1]
-including SHA-224, SHA-256, SHA-384, and SHA-512.
+Pure Rust implementation of the [SHA-2] cryptographic hash algorithms.
 
-[Documentation][docs-link]
+There are 6 standard algorithms specified in the SHA-2 standard: 
+[`Sha224`], [`Sha256`], [`Sha512_224`], [`Sha512_256`], [`Sha384`], and [`Sha512`].
```

Comment:
These are dangling links when rendered on GitHub

Reply:
Hm... I wonder how it's better to handle those. Simply remove linking? Adding explicit links would be... annoying.